www/templates/diary/infinite_scroll_list.html

```diff
@@ -51,48 +53,48 @@ <h3 ng-if="!infScrollControl.reachedEnd">{{'diary.scroll-to-load-more' | transla
         If you want to change this from collection-repeat to ng-repeat, you
         MUST test on android first to make sure that the scrolling works.
         -->
-		<div collection-repeat="trip in data.displayTrips" item-height="itemHt + 10">
-            <div class="start-time-tag-inf-scroll">{{trip.display_start_time}}</div>
-            <div  style="padding-left: 19%;">
+		<div ng-repeat="trip in data.displayTrips">
+            <div>  <!-- style="padding-left: 5%;"-->
+
+                <div ng-show="$index==0">
+                    <h2 class="hr-lines" style="justify-content: center; 
+                            font-family: 'Lato';
+                            font-weight: bold; 
+                            display: flex;
+                            color: #212121;
+                            font-size: 19px;
+                            margin-left: 10px; 
+                            margin-right: 10px;
+                            margin-bottom: 5px; 
+                            margin-top: 5px" translate> {{trip.display_date}}</h2>
+                </div>
```

Comment:
I had actually created a directive for this, but since there was no JavaScript behind the date breaks, I decided against it. I will move the horrible looking code to the CSS, my apologies.